lib/Service/CardService.php

```diff
@@ -615,8 +615,8 @@ public function findAllWithDue($userId) {
 	 * @throws \OCA\Deck\NoPermissionException
 	 * @throws BadRequestException
 	 */
-	public function findAssignedCards($userId) {
-		$cards = $this->cardMapper->findAssignedCards($userId);
+	public function findToMeOrNotAssignedCards($userId) {
+		$cards = $this->cardMapper->findToMeOrNotAssignedCards($userId);
```

Comment:
We actually don't use that method anywhere so we can just drop it for now, it didn't match the previous method signature as well.

Reply:
That will also fix the psalm error then